src/utils/vidosResolver.ts

```diff
@@ -26,7 +26,11 @@ export class VidosResolver implements Resolvable {
             },
         });
   
-        if (resolutionResponse.ok) return resolutionResponse.json();
+        if (resolutionResponse.ok) {
+            return resolutionResponse.json();
+        } else if (resolutionResponse.headers.get("content-type")?.includes("application/json")) {
```

Comment:
Ahhhh also so close, this is also potentially not valid as there are going to be some add response content types. Suggest looking for element `didResolutionResult`